recipe/run_conda_forge_build_setup_win.bat

```diff
@@ -91,15 +91,23 @@ for %%i in ("%~dp0.") do set "SCRIPT_DIR=%%~fi"
 <cuda.version set /p CUDA_VERSION=
 del cuda.version
 if not "%CUDA_VERSION%" == "None" (
-    call "%SCRIPT_DIR%\install_cuda.bat" %CUDA_VERSION%
-    if errorlevel 1 (
-        echo Could not install CUDA
-        exit 1
+    if "%CUDA_VERSION:~0,2%" == "12" (
```

Comment:
btw I tried to make this line more robust
```suggestion
    for /f "delims=." %%f in ("%CUDA_VERSION%") do (set /a "CUDA_MAJOR_VERSION=%%f")
    if %CUDA_MAJOR_VERSION% geq 12 (
```
but for some reason the CI just hated the change (it worked fine locally), so I didn't use it here... Like John said, in the future we'll drop most of the complex logic here, so I hope this is OK.